src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_Call.cs

```diff
@@ -820,6 +978,39 @@ private static ImmutableArray<RefKind> GetRefKindsOrNull(ArrayBuilder<RefKind> r
             }
         }
 
+        private static readonly DiagnosticBag s_notUsedDiagnosticBagForDerivingArgumentsForIOperation = new DiagnosticBag();
+
+        private static void AppendMissingOptionalArguments(SyntaxNode syntax,
+            ImmutableArray<ParameterSymbol> missingParameters,
+            Binder binder,
+            ArrayBuilder<BoundExpression> argumentsBuilder,
+            ArrayBuilder<ArgumentKind> argumentKindsBuilder,
+            ArrayBuilder<ParameterSymbol> parametersBuilder,
+            ThreeState enableCallerInfo = ThreeState.Unknown)
+        {
+            foreach (var parameter in missingParameters)
+            {
+                if (parameter.IsParams)
```

Comment:
>parameter.IsParams [](start = 20, length = 18)

This condition is not accurate. It should match the logic used by lowering, i.e. this is an expanded form and the parameter is the last one.